src/cli/serve.jl

```diff
@@ -221,8 +221,9 @@ function escher_serve(port=5555, dir="")
     # App
     @app static = (
         Mux.defaults,
-        route("assets", Mux.files(Pkg.dir("Escher", "assets")), Mux.notfound()),
-        route("pkg/:pkg", packagefiles( "assets"), Mux.notfound()),
+        route("escher/assets", Mux.files(Pkg.dir("Escher", "assets")), Mux.notfound()),
+        route("assets", Mux.files(dir), Mux.notfound()),
+        route("pkg/:pkg", packagefiles("escher/assets"), Mux.notfound()),
```

Comment:
This line would force other packages to have $PKGDIR/$PKG/escher/assets vs $PKGDIR/$PKG/assets - is that intended ?